.eslintrc.json

```diff
@@ -26,12 +26,14 @@
     "eslint-plugin-import",
     "eslint-plugin-jsdoc",
     "eslint-plugin-react",
+    "etc",
```

Comment:
Just wondering what this etc plugin is?

Reply:
That one's [eslint-plugin-etc](https://github.com/cartant/eslint-plugin-etc). It provides a bunch of general-purpose TypeScript-specific rules. We use it specifically for two rules:
- `etc/no-commented-out-code`: It disallows code that's been commented while allowing inline comments and JSDoc comments
- `etc/throw-error`: It disallows throwing anything that's not an instance of `Error`